src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -22,6 +22,7 @@
 use Symfony\Component\Yaml\Exception\ParseException;
 use Symfony\Component\Yaml\Parser as YamlParser;
 use Symfony\Component\ExpressionLanguage\Expression;
+use Symfony\Component\Yaml\Yaml;
```

Comment:
let's group yaml lines now that we can (but we don't move existing lines to reduce merge conflicts)